### DISCUSSION TITLE
Fast Free WebDav Server

### DISCUSSION THREAD

**Discussion by joaochris** [sircharlo/meeting-media-manager#184] [Q&A] [answered]
I was using the TeraCLOUD server but it takes a long time to download the files. Could you recommend me a faster one?

**Comment by sircharlo** [comment, marked as answer]
Good question. If hosting your own server is out of the question, then perhaps this comparison could be useful to you. I tested uploading and downloading a 100MB file using JWMMF, with the following 6 providers.


| Provider | Upload speed (Mbps) | Download speed (Mbps) | WebDAV URL | Default folder |
|-|-|-|-|-|
| 4shared | 19.98 | 39.88 | webdav.4shared.com | / |
| Box | 20.32 | 113.38 | dav.box.com | /dav/ |
| Koofr | 17.42 | 68.80 | app.koofr.net | /dav/Koofr/ |
| pCloud | 8.39 | 43.52 | webdav.pcloud.com | / |
| PowerFolder | 17.59 | 96.29 | my.powerfolder.com | /webdav/ |
| TeraCloud | 9.77 | 31.83 | seto.teracloud.jp | /dav/ |

As you can see, **Box**, **4shared** and **PowerFolder** seem to be the clear winners here. TeraCloud is the slowest of them all.

**Reply by joaochris** [reply]
Excellent brother! I just tested it with BOX and it really is very fast. Thank you for your help!

**Reply by sircharlo** [reply]
For those reading this now: pCloud has just announced that WebDAV will only be supported for premium users as of next week. As a result, pCloud will be removed from the list of suggested providers in the next release.

**Reply by Chipperchap** [reply]
I started by using 4Shared but it just would not let me connect no matter what I did.  I suspect it was being blocked somehow by my PC firewall or similar as a virus risk "Cloud", so I signed up for PowerFolder instead and it worked instantly, glitch free and has been fine over the last day or so. (I only learned about this software two days ago). It doesn't take too long to sync so hopefully it continues to behave as I have been asked by all the ZOOM Hosts in our congregation to show them JWMMF and trial it on their PC's.

**Comment by riemsesy** [comment]
fyi
From the Box website. So if it works it works if it doesn't it doesn't
Hi,

I am the Product Manager for Box Uploads & Downloads and would like to give some clarity about WebDav support at Box.

As some of you noticed, this sync method has been out-of-support for close to 3 years now and we strongly advise users and customers to not use WebDav for new applications and use-cases but use our regular APIs instead: we maintain very basic support of WebDav (mostly security-wise) and will likely end-of-life WebDav next year. If you have existing applications that leverage WebDav it would be a good idea to start planning for a migration to our REST APIs.

While I imagine this could be disappointing news, I hope this helps you all  the right development decisions.

Jacques

**Comment by e-Feitosa** [comment]
I use **Google Drive**(I downloaded Google Drive for PC), and it has worked relatively well. I installed it on my PC and on the Salon (KH) using the same account and email. I created some folders and in the M³ I chose the folder corresponding to the one in the Salon (KH). At home I download the media, and the other PC already updates normal. I do not know if it would be ideal or could be improved, but it has been useful to send the images that the speaker will use in the speech at the weekend.

![image](https://user-images.githubusercontent.com/90984283/194095721-95431500-4f9a-4636-82a1-f9a50ecc115f.png)

**Reply by mtdvlpr** [reply]
The only thing you are missing by using this setup, is the auto rename features. When you change your date format or media language. The folders and song/paragraph files will be automatically renamed to the new format/language. This doesn't work for you, because M3 does not have permissions to do this I have noticed.
